constructor/__init__.py

```diff
@@ -1,3 +1,5 @@
-from . import _version
-
-__version__ = _version.get_versions()['version']
+try:
+    from . import _version
+    __version__ = _version.__version__
+except ImportError:
+    __version__ = "dev"
```

Comment:
I like this [idiom](https://github.com/conda/conda-libmamba-solver/blob/c25621abd4e4b0d9726223c617eaef5fcfa22807/conda_libmamba_solver/__init__.py#L4-L13) better:

```suggestion
try:
    from ._version import version as __version__
except ImportError:
    try:
        from importlib.metadata import version

        __version__ = version("constructor")
        del version
    except ImportError:
        __version__ = "0.0.0.unknown"
```